src/styles/utils/_variables.scss

```
@@ -0,0 +1,16 @@
+$square-width: 100px;
+$square-height: 100px;
+$square-gap: 1px;
+$week-length: 7;
+$container-width:  #{($square-width + $square-gap) * $week-length};
```

```suggestion
$container-width:  ($square-width + $square-gap) * $week-length;
```
You can to not use **#{}** here